### PR TITLE
Hackathon2

### PR DESCRIPTION
**Pull request by Free-Quarks**
This is a quick and hacky PR to expand the coverage of our EQ2AMR endpoints but using the new MathExpressionTree data structure and all the coverage additions it comes with. 

It also includes updates to the AMR spec requested by TA-4. 

(I am aware the get_terms() function in first_order_ode.rs is in desperate need of a refactoring.)

### CODE REVIEW

File: data/mml2pn_inputs/testing_eqns/mml_list_sqair.txt
Comment (cl4yton) [inline data/mml2pn_inputs/testing_eqns/mml_list_sqair.txt:1]: Delete this file

**Review by cl4yton — CHANGES_REQUESTED** [review]
Delete file, but otherwise looks good

File: data/mml2pn_inputs/testing_eqns/mml_list_sqair.txt
Comment (Free-Quarks) [reply data/mml2pn_inputs/testing_eqns/mml_list_sqair.txt:1]: Done!

**Review by cl4yton — APPROVED** [review]
Looks good to me!

**Pull request merged**